pilot/pkg/features/pilot.go

```diff
@@ -369,4 +369,8 @@ var (
 		5*time.Second,
 		"The timeout to send the XDS configuration to proxies. After this timeout is reached, Pilot will discard that push.",
 	).Get()
+
+	EndpointTelemetryLabel = env.RegisterBoolVar("PILOT_ENDPOINT_TELEMETRY_LABEL", false,
```

Comment:
When will this be set to true? in 1.8?

Reply:
When telemetry is truned on.

Reply:
so essentially every install will have this set then is what I wanted to confirm? or at least, the default, since telemetry is enabled by default

Reply:
Yes, you are right. I would like to turn this on by default, but it brings no positive value to telemetry disabled installation as telemetry filter is the only filter consumes this specially formatted metadata.

Reply:
SGTM, just wanted to make sure I was on the same page